internal/types/backup.go

```diff
@@ -120,13 +123,63 @@ func (o *TakeBackupOperation) GetMessage() string {
 func (o *TakeBackupOperation) SetMessage(m string) {
 	o.Message = m
 }
+func (o *TakeBackupOperation) GetCreatedAt() time.Time {
+	return o.CreatedAt
+}
+func (o *TakeBackupOperation) SetCreatedAt(createdAt time.Time) {
+	o.CreatedAt = createdAt
+}
+
+type RestoreBackupOperation struct {
```

Comment:
It is also needed to correctly return `RestoreBackupOperation` instances from `process_result_set.go`